test/services/static-snapshot-service.test.ts

```diff
@@ -0,0 +1,86 @@
+import {describe} from 'mocha'
+import Constants from '../../src/services/constants'
+import {StaticSnapshotOptions} from '../../src/services/static-snapshot-options'
+import StaticSnapshotService from '../../src/services/static-snapshot-service'
+import {captureStdOut} from '../helpers/stdout'
+import chai from '../support/chai'
+
+const expect = chai.expect
+
+describe('StaticSnapshotService', () => {
+  const port = Constants.PORT
```

Comment:
remove `port` and `host`, add a public `baseUrl` function to the `StaticSiteService` and then make use of that function in these tests.